software_attributes/schema.py

```diff
@@ -171,10 +171,44 @@
                 }
             ],
         },
+        "turbine_model": {
+            "description": "The fidelity with which the turbine is represented in the software model (if at all).",
+            "type": "array",
+            "items": {
+                "enum": ["regression",
+                         "lookup_table",
+                         "momentum_sink",
+                         "multibody_bem",
+                         "vortex_method",
+                         "actuator",
+                         "geometry_resolved",
+                         "SCADA",
+                         "N/A"]
+            }
+        },
+        "wake_model": {
+            "description": "The fidelity with which turbine wakes are represented in the software model (if at all). ",
+            "type": "array",
+            "items": {
+                "enum": ["analytical",
+                         "dynamic_meandering",
+                         "vortex_method",
+                         "navier_stokes",
+                         "N/A"]
+            }
+        },
         "loads": {
             "description": "Does the software calculate system loads?",
             "type": "boolean"
         },
+        "design_optimization": {
+            "description": "Can the software conduct design optimization?",
+            "type": "boolean"
+        },
+        "costs": {
+            "description": "Does the software include or focus on costs?",
+            "type": "boolean"
```

Comment:
Similar to the previous comment, costs is included in the "primary_uses" field, but maybe it's worth expanding on here.

Reply:
Same comment.  WISDEM can do design, optimization, and costs, but would have to drop two to pick just one primary use.  Worth a conversation.